src/docfx/build/toc/parser/MarkdownTocMarkup.cs

```diff
@@ -15,24 +14,26 @@ namespace Microsoft.Docs.Build
 {
     internal static class MarkdownTocMarkup
     {
-        private static readonly HashSet<Type> s_blockWhiteList = new HashSet<Type> { typeof(HeadingBlock) /*header*/, typeof(YamlFrontMatterBlock) /*yaml header*/, typeof(HtmlBlock) /*comment*/ };
```

Comment:
why need change that? we should only need add html block parser back?

Reply:
There are other types of html block types, we only support comments.